### ISSUE TITLE
Multiple entires for 'files'

### ISSUE THREAD

**Issue by logan-han** [buildkite-plugins/test-collector-buildkite-plugin#40]
We have a in-house script to push the junit tests created when the plugin wasn't available. 

It relies on the test results download via the artifacts plugin:
```
 - artifacts#v1.5.0:
        download: [ "*/build/test-results/test/TEST-*.xml", "*/build/test-results/integrationTest/TEST-*.xml" ]
```

As the plugin supports only one path for `files`, I've attempted to use `"*/build/test-results/*/TEST-*.xml"` instead but apparently it doesn't cover all the files uploaded. 

Please either make it supports `**` or just more than one path.

**Comment by nate-thirdwave** [comment]
Looking at the implementation, I think the straightforward approach would be to extend the `base-path` to accept multiple entries.  The underlying call to `find` supports multiple starting points so the logic update would be minimal.

**Comment by toote** [comment]
hi @nate-thirdwave , that is indeed a weird behaviour.

From what I understand you are downloading artifacts from 2 different paths: `FOLDER_1/build/test-results/test` and `FOLDER_2/build/test-results/integrationTest`. You then use the plugin in a step as follows:

```yaml
- command: true
  plugins:
    - artifacts#v1.5.0:
        download: [ "*/build/test-results/test/TEST-*.xml", "*/build/test-results/integrationTest/TEST-*.xml" ]
    - test-collector#v1.6.0:
        files: "*/build/test-results/test/*/TEST-*.xml"
```

If that is the case, my tests show that it should work as expected:

```
$ mkdir -p FOLDER_1/test-results/test/
$ mkdir -p FOLDER_2/test-results/integrationTests/
$ touch FOLDER_2/test-results/integrationTests/file1.xml
$ touch FOLDER_2/test-results/integrationTests/file2.xml
$ touch FOLDER_1/test-results/test/file2.xml
$ touch FOLDER_1/test-results/test/file1.xml
$ find . -path './*/test-results/*/*.xml'
./FOLDER_2/test-results/integrationTests/file2.xml
./FOLDER_2/test-results/integrationTests/file1.xml
./FOLDER_1/test-results/test/file2.xml
./FOLDER_1/test-results/test/file1.xml
```

Or did I misinterpret any part of your scenario?

**Comment by toote** [comment]
Makes sense... let me know if that is something you want to try to create a PR on

In the meantime, as a workaround and if you can't or won't generalize the file pattern, you can also just add the plugin to the step twice and get the same effect the following way:

```yaml
- command: true
  plugins:
    - artifacts#v1.5.0:
        download: [ "*/build/test-results/test/TEST-*.xml", "*/build/test-results/integrationTest/TEST-*.xml" ]
    - test-collector#v1.6.0:
        files: "*/build/test-results/test/TEST-*.xml"
    - test-collector#v1.6.0:
        files: "*/build/test-results/integrationTest/TEST-*.xml"
```

**Comment by toote** [comment]
Note that we have implemented your suggestion (in #46) and released 1.7.0 with the new functionality :)
